src/app/content/components/Page/contentLinkHandler.spec.ts

```diff
@@ -152,6 +152,7 @@ describe('contentLinkHandler', () => {
 
     it('intercepts clicking content links with slug', async() => {
       const link = `/books/${book.slug}/pages/page-title`;
+      prop.currentPath = '/asdf?foo=bar';
```

Comment:
Is this unrelated? This seems to be the only instance in the PR of something changing that has to do with currentPath.

Reply:
I modified the test so that I could have one with a pre-existiing search string.

Reply:
I further modified the test to check the search string result.